MyCryptoHQ/MyCrypto
# Fixed gas price Bug

### Description

Gas price bug first pointed out here: https://twitter.com/milodinoX/status/1100924940520370176

### Changes

* Set a cap on number of characters for a gas price, but also for number of decimal places.

### Steps to Test

1. Access wallet, try to send with gas price of: `10.0000000000000010` Shouldn't allow you to send.
2. Try to set a gas price of `2999.99999`, that should work.
3. Try to set a gas price of `2999.9999999`, that should fail.
